### DISCUSSION TITLE
Canali, abilitazione commenti

### DISCUSSION THREAD

**Discussion by maelaborazioni** [globaleaks/globaleaks-whistleblowing-software#3873] [Q&A]
Buonasera,
ho notato che quando rientro nelle impostazioni di una canale, i dati vengono salvati correttamente ma se clicco su Avanzate, i flag proposti non sono quelli che avevo salvato la volta precedente. In particolare quello relativo ai commenti me lo ritrovo sempre non flaggato quando rientro... Ho la versione 4.13.18. Upgradando la versione andrà a posto? Chiedo perchè al momento, se aggiorno, il sito mi diventa non più raggiungibile e quindi sto aspettando a metter su l'ultima release.
Grazie

**Comment by evilaliv3** [comment]
Circa il possibile bug che segnala verifichiamo e le facciamo sapere.  

In merito all'aggiornamento dovrebbe già poter aggiornare all'ultima versione ed il sito dovrebbe essere raggiungibile; l'aggiornamento però non contiene nessuna variazione rispetto a quanto riportato per le configurazioni dei canali.

**Reply by maelaborazioni** [reply]
D'accordo, grazie mille intanto!
